docs/dev/explanations/components/index.md

```diff
@@ -128,7 +129,7 @@ See this diagram for an example of how this looks in practice:
 TODO: document the entry point
 
 - `diracx-routers`:
-  - `diracx.diracx_min_client_version` entry-point defines the diracx minimum client version required by the server to prevent issues. This also searches for extension names instead of `diracx`. The minimum version number has to be updated in `diracx-routers/src/__init.py__`
+    - `diracx.diracx_min_client_version` entry-point defines the diracx minimum client version required by the server to prevent issues. This also searches for extension names instead of `diracx`. The minimum version number has to be updated in `diracx-routers/src/__init.py__`
```

Comment:
```suggestion
    - `diracx.diracx_min_client_version` entry-point defines the DiracX minimum client version required by the server to prevent issues. This also searches for extension names instead of `diracx`. The minimum version number has to be updated in `diracx-routers/src/__init.py__`
```

and specify what is an acceptable extension name.